src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <div class="card__picture_container">
+        <img
+          class="card__picture"
+          src="./images/imac.jpeg"
+          alt="Product picture"
+        >
+      </div>
+
+      <h3 class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h3>
+
+      <p class="card__item-number">Код товара: 195434</p>
+
+      <div class="card__star-rating_container">
+        <ul class="card__star-rating_list">
```

Comment:
use either `block-name__element-name` or `blcok_name__element_name`, not both

Reply:
ok, thanks )